lib/publiccloud/instance.pm

```diff
@@ -364,6 +364,7 @@ Parameters:
  timeout => total wait timeout; default: 600.
  wait_stop => If true waits for ssh port to become unreachable, if false waits for ssh reachable; default: false.
  proceed_on_failure => in case of fail, if false exit test with error, if true let calling code to continue; default: wait_stop.
+ scan_ssh_host_key => If true we will rescan the SSH host key
```

Comment:
default is 0. Could you expand here the sentence listing some situations where `scan_ssh_host_key => 1` should be used?

Reply:
Done.